app/controllers/application_controller.rb

```diff
@@ -79,6 +79,7 @@ def make_tag_suffix(link, sanitized_blacklist, query)
       kink_in_query = query.split(' ').any? { |tag| kinks.any? tag }
     end
     append_to_tags = ''
+    append_to_tags = '-flash'
```

Comment:
```suggestion
    append_to_tags = '-flash '
```

since this is appended to the theme tag, we need a space to make sure it's not going to combine with this tag